plugins/org.python.pydev.refactoring/src/org/python/pydev/refactoring/coderefactoring/inlinelocal/edit/RemoveAssignment.java

```diff
@@ -56,19 +59,21 @@ public RemoveAssignment(InlineLocalRequest req) {
             boolean anotherStatementOnThisLine = false;
 
             /* now find the end */
+            FastPartitioner fastPartitioner = (FastPartitioner) PyPartitionScanner.checkPartitionScanner(document);
             while (true) {
-                char c = document.getChar(endOffset - 1);
```

Comment:
The fix seems ok for me, but this should have a test-case along with it.

i.e.: See the files in `python.coderefactoring.inlinelocal` which are run from the `org.python.pydev.refactoring.tests.coderefactoring.inlinelocal.InlineLocalTestSuite` (from `org.python.pydev.refactoring/tests`).